en/guide/behind-proxies.md

```diff
@@ -74,7 +74,7 @@ Enabling `trust proxy` will have the following impact:
   </li>
   <li markdown="1">`X-Forwarded-Proto` can be set by the reverse proxy to tell the app whether it is `https` or  `http` or even an invalid name. This value is reflected by [req.protocol](/{{ page.lang }}/api.html#req.protocol).
   </li>
-  <li markdown="1">The [req.ip](/{{ page.lang }}/api.html#req.ip) and [req.ips](/{{ page.lang }}/api.html#req.ips) values are populated with the list of addresses from `X-Forwarded-For`.
+  <li markdown="1">The [req.ip](/{{ page.lang }}/api.html#req.ip) value is got from the left-most entry in the `X-Forwarded-For` header, while [req.ips](/{{ page.lang }}/api.html#req.ips) is populated with the list of addresses from `X-Forwarded-For`.
```

Comment:
Great catch! In fact, this is still incorrect, as in this part "enabled" means anything except `false`, so it is about `true`, `2`, `10.0.0.1/16`, etc. values. Many times it will not be the left-most entry (only in the case of `true`, which is only one of the enabled states).

Reply:
Thanks, you clarified another doubt I had.

This passage talks of "enabling", but earlier in the doc it seems that this behaviour is attained when it is exactly `true` (see [the first table entry here](https://expressjs.com/en/guide/behind-proxies.html)):


Type | Value
-- | --
Boolean | If `true`, the client’s IP address is understood as the left-most entry in the X-Forwarded-* header. If `false`, the app is understood as directly facing the Internet and the client’s IP address is derived from req.connection.remoteAddress. This is the default setting.

What's the best phrasing to choose?

I am no Express pro, sorry if I may not being precise. Thanks again.

Reply:
You're right, the page does not define enabling, as it just uses the term from how app settings work-- any value for a setting that is being enabled. The call of `app.set` or `app.enable` enables a setting while `app.disable` disables a setting.

Reply:
We should improve the wording somehow--this section is supposed to explain what APIs of Express are altered by changing the "trust proxy" setting, if that makes sense--no matter what value it is set to.

Reply:
> We should improve the wording somehow

No native speaker, sorry :)